UnityProject/Assets/Scripts/HealthV2/Living/PlayerHealthV2.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Collections;
+using System.Collections.Generic;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6324510)